src/styles/main.scss

```diff
@@ -1,3 +1,71 @@
+$start-day-values:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+$cell-size: 100px;
+$gap: 1px;
+$calendar-padding: 10px;
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  gap: 1px;
+  width: (7 * $cell-size + 6 * $gap + 2 * $calendar-padding);
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-padding;
+
+  &__day {
+    border: 1px solid #000;
+    width: 100px;
+    height: 100px;
```

Comment:
Replace the fixed height (100px) with the variable $cell-size to maintain consistency.